scripts/summit/comCam/runIsrRunner_000.py

```diff
@@ -30,10 +30,10 @@
 
 location = 'slac' if len(sys.argv) < 2 else sys.argv[1]
 locationConfig = LocationConfig(location)
-butler = dafButler.Butler(locationConfig.ts8ButlerPath, collections=['LSST-TS8/raw/all', 'LSST-TS8/calib'])
+butler = dafButler.Butler(locationConfig.comCamButlerPath, collections=['LSSTComCam/raw/all', 'LSSTComCam/calib'])
```

Comment:
This commit https://github.com/lsst-sitcom/rubintv_production/pull/48/commits/9c733b0e873c4506cb829f5b7f0028a08ef36a7f should squash with https://github.com/lsst-sitcom/rubintv_production/pull/48/commits/a55b72c0d757851eb7e0195d8e906d3eb60eb941